README.md

```diff
@@ -47,7 +47,7 @@ Figure out what you need to change to give Javascript access to the `h1#header`
 
 **YOUR NOTES**
 ```
-
+Needed to use window.onload, which makes sure all elements have loaded before getting the header.
 ```
```

Comment:
Other options include using the `defer` attribute on the script tag or using `DOMContentLoaded`. Slight variations among all of them, but they will all work here. Here's an interesting comparison between load and DomContentLoaded! http://web.archive.org/web/20150405114023/http://ie.microsoft.com/testdrive/HTML5/DOMContentLoaded/Default.html